tests/e2e/pageobjects/ide/Ide.ts

```diff
@@ -218,32 +216,29 @@ export class Ide {
         await this.waitStatusBarContains(expectedTextInStatusBar, 20000);
     }
 
-    async closeAllNotifications() {
-        const notificationLocator: By = By.css('.theia-Notification');
-
+    async closeAllNotifications(timeout: number = TestConstants.TS_SELENIUM_DEFAULT_TIMEOUT) {
+        const statusBarNotificationsLocator: By = By.xpath(`//div[@id="theia-statusBar"]/div[@class="area right"]//div[contains(@title, 'Notification')]`);
+        const notificationCenterActionCloseAllButtonLocator: By = By.xpath(`//div[contains(@class, 'theia-notifications-container theia-notification-center open')]//ul[@class="theia-notification-actions"]/li[@class="clear-all"]`);
+        const notificationCenterClosedLocator: By = By.xpath(`//div[contains(@class, 'theia-notifications-container theia-notification-center closed')]`);
+        const notificationCenterContainerLocator: By = By.xpath(`//div[contains(@class, 'theia-notifications-container theia-notification-center')]//div[@class='theia-notification-list']`);
         Logger.debug('Ide.closeAllNotifications');
 
-        if (! await this.driverHelper.isVisible(notificationLocator)) {
-            return;
-        }
-
-        const notifications: WebElement[] = await this.driverHelper.waitAllPresence(notificationLocator);
-        const notificationsCapacity: number = notifications.length;
-
-        for (let i = 1; i <= notificationsCapacity; i++) {
-            const notificationLocator: By = By.xpath('//div[@class=\'theia-Notification\']//button[text()=\'Close\']');
-
-            try {
-                await this.driverHelper.waitAndClick(notificationLocator);
-            } catch (err) {
-                if (err instanceof error.TimeoutError) {
-                    console.log(`The '${notificationLocator}' element is not visible and can't be clicked`);
-                    continue;
-                }
-
-                throw err;
+        Logger.trace(`Ide.closeAllNotifications driverHelper click Notification Center icon.`);
```

Comment:
For me looks too congested. In my opinion if delete all "Logger.trace" occurrences the code will look much clearer, readable and understandable.

Reply:
well, before you suggested to switch it to trace so that it doesn't pollute the debug. Traces are really valuable for me sometimes when I'm trying to figure out what went wrong.

If you're ok with that, I would prefer to keep it there

Reply:
I would argue it makes the code even more understandable.

Reply:
@ScrewTSW: `Logger.trace(`Ide.closeAllNotifications driverHelper click Notification Center icon.`);` duplicates `Logger.debug('Ide.closeAllNotifications');` and so looks redundant in final code.
It might help in development of tests, but make it harder to support by others. 
Actually, it's the propose of comments and proper naming of classes/variables/methods to make the code more understandable - logging command makes code execution more understandable, but the price is less understandable code which harder to read.